cloudbuild.yaml

```diff
@@ -65,7 +65,9 @@ steps:
         -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
         -var=autopilot_cluster=$_AUTOPILOT_CLUSTER \
         -var=cluster_location=$_REGION \
-        -auto-approve -no-color -lock=false
+        -var='cpu_pools=[{initial_node_count=2,name="cpu-pool",machine_type="n1-standard-16",autoscaling=true,min_count=1,max_count=3,disk_size_gb=100,disk_type="pd-standard",}]' \
```

Comment:
We have dedicated tfvars files for tests, slightly cleaner to just add this change directly into https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/infrastructure/tfvars_tests/standard-gke-public.platform.tfvars